README.md

```diff
@@ -55,7 +55,9 @@ steps:
 
 ## Properties
 
-* `files` — Required — String — Pattern of files to upload to Test Analytics
+* `files` — Required — String — Pattern of files to upload to Test Analytics.  May contain `*` which matches on any
+  number of characters of any type.  Note that `/` and `.` are not treated specially.  The pattern will be prepended
+  with `./` and searching will start from the root of the checkout.
```

Comment:
I agree that more clarifications are necessary. What do you think of this alternative wording?

```suggestion
* `files` — Required — String — Pattern of files to upload to Test Analytics, relative to the checkout path (`./` will be added to it). May contain `*` to match any number of characters of any type (unlike shell expansions, it will match `/` and `.` if necessary)
```